### PR TITLE
[PE-4856] Create -inverse table version

### PR DESCRIPTION
**Pull request by jllr**

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e117de22441e72a5391078340d4f31500ca5e93d-chi-pr.enterprise.ctl.io" target="_blank"> https://e117de22441e72a5391078340d4f31500ca5e93d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/951/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://6d5525f5cbc5b65b34b449005e5629631d78cc0c-chi-pr.enterprise.ctl.io" target="_blank"> https://6d5525f5cbc5b65b34b449005e5629631d78cc0c-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/955/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ca94d2cbb2c12b773dcdfe93dd4ad63bd8a237c4-chi-pr.enterprise.ctl.io" target="_blank"> https://ca94d2cbb2c12b773dcdfe93dd4ad63bd8a237c4-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/956/ for more details.

File: src/chi/components/tables/tables.scss [hunk 1]
```diff
@@ -146,6 +169,101 @@ th {
     }
   }
 
+  &.-inverse {
+    thead {
+      tr {
```
Comment (mattnickles) [inline src/chi/components/tables/tables.scss:174]: Need to define a background color on the tr as it's inheriting white from `.chi .chi-table tr`

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/966/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://520985096fc6787624eefa52134dd1c9cbf6d25d-chi-pr.enterprise.ctl.io" target="_blank"> https://520985096fc6787624eefa52134dd1c9cbf6d25d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/967/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-zwdgo0, https://ops.elasticbox.com/services/instances/i-yj9gkc, https://ops.elasticbox.com/services/instances/i-mzxiwm, https://ops.elasticbox.com/services/instances/i-56ra11, https://ops.elasticbox.com/services/instances/i-fg473f

**Pull request merged**